### PR TITLE
Add new template

### PR DESCRIPTION
**Pull request by sovrasov**
Also added summary to existing templates

### CODE REVIEW

**Comment by sovrasov** [comment]
@Ilya-Krylov please take a look at the new summaries.

File: configs/ote_custom_classification/efficientnet_v2_s/template_experimental.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Description.
+model_template_id: Custom_Image_Classification_EfficientNet-V2-S
+name: EfficientNet-V2-S
+task_type: CLASSIFICATION
+task_family: VISION
+instantiation: "CLASS"
+summary: This model is quite slow, but provides superior single and multi label classification performance.
```
Comment (kprokofi) [inline configs/ote_custom_classification/efficientnet_v2_s/template_experimental.yaml:7]: I wouldn't call it slow, slower than others (in our templates) in comparison - yes, but generally, it is not so slow, like ResNet50 +-

File: configs/ote_custom_classification/efficientnet_b0/template.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ name: EfficientNet-B0
 task_type: CLASSIFICATION
 task_family: VISION
 instantiation: "CLASS"
-summary: Custom Image Classification EfficientNet-B0
+summary: Provides better performance on large datasets, but may be not so stable in case of small amount of training data.
```
Comment (kprokofi) [inline configs/ote_custom_classification/efficientnet_b0/template.yaml:7]: Do we keep efficientnetb0 for v1.1?  About a small amount. I would consider providing the lower boundary for this small amount (maybe 300-400?)

File: torchreid/utils/torchtools.py [hunk 3]
```diff
@@ -384,18 +384,13 @@ def load_pretrained_weights(model, file_path='', chkpt_name='model_weights', pre
...
-        return key
-
     def _remove_prefix(key, prefix):
         prefix = prefix + '.'
         if key.startswith(prefix):
             key = key[len(prefix):]
         return key
 
     is_file = check_isfile(file_path)
-    pretraining_timm_model = model.pretrained and isinstance(model, TimmModelsWrapper)
```
Comment (kprokofi) [inline torchreid/utils/torchtools.py:398]: If we remove this - we can't load weights for timm models provided through link

Comment (sovrasov) [reply torchreid/utils/torchtools.py:398]: That depends on the link we provide. Downloading the official timm weights has no sense in this case: we will use custom link and custom snapshot with correct layer names.

File: configs/ote_custom_classification/efficientnet_b0/template.yaml (hunk 2 above)
Comment (sovrasov) [reply configs/ote_custom_classification/efficientnet_b0/template.yaml:7]: I guess no. At the moment procedure of deleting model templates is not well established, so I can't do it right now. I don't know who will read this description and can we add any exact numbers here. @Ilya-Krylov what do you think?

File: configs/ote_custom_classification/efficientnet_v2_s/template_experimental.yaml (hunk 1 above)
Comment (sovrasov) [reply configs/ote_custom_classification/efficientnet_v2_s/template_experimental.yaml:7]: For an SC user these 2 templates is the whole world, so I guess that's true at least at this scale. This template is noticeably slower than MoblieNets.

**Review by alexsu52 — COMMENTED** [review]
Before including efficientnet_v2_s in the SC scope, this model must be supported in the optimization pipeline.

**Comment by sovrasov** [comment]
@alexsu52 the new model template is in experimental status now, that means it is not visible for SC.

File: configs/ote_custom_classification/efficientnet_b0/template.yaml (hunk 2 above)
Comment (Ilya-Krylov) [reply configs/ote_custom_classification/efficientnet_b0/template.yaml:7]: I think we should discuss it first

**Review by Ilya-Krylov — APPROVED** [review]

**Pull request merged**